Cargo.toml

```diff
@@ -1,15 +1,18 @@
 [package]
-name = "barcodex-rs"
-version = "0.1.2"
 authors = ["Andre Masella <andre.masella@oicr.on.ca>"]
 edition = "2018"
-
+name = "barcodex-rs"
+version = "0.1.3"
 [dependencies]
 bio = "^0.31"
 bio-types = "^0.7"
-clap ="3.0.0-beta.1"
+clap = { version = "^3.1", features = ["derive"]}
+derive = "*"
 flate2 = "^1.0"
 lazy_static = "^1.4"
 regex = "^1.3"
-serde = { version = "^1.0", features = ["derive"]}
 serde_json = "^1.0"
+
+[dependencies.serde]
```

Comment:
Why change to this format? The new `clap` line uses the old format and the official `sarde` documentation used the old format (https://serde.rs/derive.html)

Reply:
I don't think that I edited this file by hand. I just used the `cargo` tool to update, add and remove dependencies.